.github/workflows/actions.yml

```diff
@@ -2,20 +2,54 @@ name: ci
 
 on:
   push:
+    branches:
+      - master
+    tags:
+      - '*'
   pull_request:
     branches:
       - master
 
 jobs:
-  test:
+  test-js:
+    runs-on: ubuntu-latest
+    env:
+      JAVA_OPTS: "-Xss10M"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Run JS tests
+        run: |
+          ./mill -i __.resolvedIvyDeps
+          ./mill -i -j $(nproc) all __.js.__.compile __.js.__.publishArtifacts __.js.__.test
```

Comment:
Here I want to run in parallel because the CI is slow.
To avoid the Coursier problems when running in parallel I first run `__.resolvedIvyDeps` without `-j $(nproc)` so after that all the dependencies are already downloaded.
I tried to use `__.prepareOffline` instead of `__.resolvedIvyDeps` but it fails on `resolvedAmmoniteIvyDeps` for Scala js `0.6` and Scala `2.11`. Probably Ammonite dropped these versions long time ago.